docs/provision.md

```diff
@@ -15,10 +15,10 @@ reboot:
 
 2. Build a box
 
-    - Option 1: Use the `centos7-provision-nightly` box.
+    - Option 1: Use the `centos9-provision-nightly` box.
 
-    - Option 2: Use an existing katello box (e.g. centos7-katello-p4-nightly) and run the provisioning playbook (it takes a while, as it syncs (on-demand) centos7, puppet 4).   ***Note*: If you are using puppet 4, you need to increase the ram on the box, to something like 8096 otherwise candlepin crashes with OOM.**
-        `ansible-playbook -l centos7-katello-p4-nightly playbooks/katello_provisioning.yml`
+    - Option 2: Use an existing katello box (e.g. centos9-katello-p4-nightly) and run the provisioning playbook (it takes a while, as it syncs (on-demand) centos9, puppet 4).   ***Note*: If you are using puppet 4, you need to increase the ram on the box, to something like 8096 otherwise candlepin crashes with OOM.**
+        `ansible-playbook -l centos9-katello-p4-nightly playbooks/katello_provisioning.yml`
```

Comment:
We haven't supported Puppet 4 for a long time.
```suggestion
    - Option 2: Use an existing katello box (e.g. centos9-katello-p7-nightly) and run the provisioning playbook (it takes a while, as it syncs (on-demand) centos9, puppet 7).   ***Note*: If you are using puppet 7, you need to increase the ram on the box, to something like 8096 otherwise candlepin crashes with OOM.**
        `ansible-playbook -l centos9-katello-p4-nightly playbooks/katello_provisioning.yml`
```